packages/datagateway-download/src/downloadStatus/adminDownloadStatusTable.component.tsx

```diff
@@ -0,0 +1,600 @@
+import React, { useCallback, useMemo } from 'react';
+import {
+  CircularProgress,
+  Grid,
+  IconButton,
+  LinearProgress,
+  Paper,
+  Typography,
+} from '@material-ui/core';
+
+import {
+  DateColumnFilter,
+  Download,
+  formatBytes,
+  FormattedDownload,
+  Order,
+  Table,
+  TableActionProps,
+  TextColumnFilter,
+} from 'datagateway-common';
+
+import { useTranslation } from 'react-i18next';
+import { IndexRange, TableCellProps } from 'react-virtualized';
+import { DownloadSettingsContext } from '../ConfigProvider';
+import {
+  adminDownloadDeleted,
+  adminDownloadStatus,
+  fetchAdminDownloads,
+} from '../downloadApi';
+import {
+  PauseCircleFilled,
+  PlayCircleFilled,
+  RemoveCircle,
+  Restore,
+} from '@material-ui/icons';
+import RefreshIcon from '@material-ui/icons/Refresh';
+import BlackTooltip from '../tooltip.component';
+
+const AdminDownloadStatusTable: React.FC = () => {
+  // Load the settings for use
+  const settings = React.useContext(DownloadSettingsContext);
+  // Sorting columns
+  const [sort, setSort] = React.useState<{ [column: string]: Order }>({});
+  const [filters, setFilters] = React.useState<{
+    [column: string]:
+      | { value?: string | number; type: string }
+      | { startDate?: string; endDate?: string };
+  }>({});
+  const [data, setData] = React.useState<FormattedDownload[]>([]);
+  const [dataCount, setDataCount] = React.useState<number>(-1);
+  const [dataLoaded, setDataLoaded] = React.useState(false);
+  const [refreshDownloads, setRefreshDownloads] = React.useState(false);
+  const [lastChecked, setLastChecked] = React.useState('');
```

Comment:
I think we should initialise this to be the current time as the last checked UI looks weird without a timestamp on initial load.